src/transformState.js

```diff
@@ -6,6 +6,27 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
you need `default` case here, where you can throw an error, so user can know, that he gave an unknown property

Reply:
added default case